options/locale/locale_en-US.ini

```diff
@@ -27,6 +27,9 @@ licenses = Licenses
 return_to_gitea = Return to Gitea
 more_items = More items
 
+guide.welcome = Welcome to the Gitea.
+guide.welcome_desc = Git with a cup of tea! Painless self-hosted all-in-one software development service, including Git hosting, code review, team collaboration, package registry and CI/CD.
```

Comment:
I think it is better to merge `APP_NAME` here, for giving user more freedom of changing the word `Gitea` here and the description.
We have `APP_NAME` in app.ini, but it seems that it is too long in here.
Maybe we can add `APP_DESCRIPTION` or something else.
You can also find the default value of  `startpage.app_desc` is a part of `guide.welcome_desc`